vita3k/renderer/include/renderer/state.h

```diff
@@ -89,8 +89,8 @@ struct State {
     void set_surface_sync_state(bool disable) {
         disable_surface_sync = disable;
     }
-    void set_stretch_display(bool disable) {
-        stretch_the_display_area = disable;
+    void set_stretch_display(bool state) {
+        stretch_the_display_area = state;
```

Comment:
Give it a meaningful name like stretch (also in the header) and not state

Reply:
State is perfect, why stretch ? Is strange
Is bool indicate litteraly state ?

Reply:
state is perfect? If I call set_stretch_display with state = false, are you able to deduce what it does? Does it stretch the display or not?

Reply:
I no understund ? 
State false = disable ? So is clear 🤔 I no understund what you wabt said here 🤷‍♂️

Reply:
done